setup-le.sh

```diff
@@ -1,14 +1,29 @@
 #!/usr/bin/bash
 set -o nounset -o errexit
 
+sed -i "s/server.example.test/`hostname -f`/g"
```

Comment:
Is this intended to change the one line in ipa-httpd.cnf?

Reply:
Correct!  Just hoping to make this a bit simpler.

Reply:
It is outdated, as i've now used the FQDN variable instead of calling `hostname -f`